### ISSUE TITLE
arch to image mapping with nodeSelector

### ISSUE THREAD

**Issue by pokebadgerswithspoon** [influxdata/telegraf-operator#67]
`telegraf-operator` is an amd64 image, `telegraf` supports multiarch

Would it be a good idea to add nodeSelector to telegraf-operator deployment part of https://github.com/influxdata/telegraf-operator/blob/master/deploy/dev.yml ?

Like this
```
    spec:
      serviceAccountName: telegraf-operator
      nodeSelector:
        kubernetes.io/arch: amd64      
```

So operator itself would require amd64 part of a cluster.

##### What products and version are you using?

telegraf-operator:1.3

**Comment by wojciechka** [comment]
Thank you for reporting this and I apologize for the late response.

That's a very good point. We'll look into changing our build pipelines so the `telegraf-operator` image is multi-arch. If not, we'll look into adding `nodeSelector`.

**Comment by wojciechka** [comment]
> Thank you for reporting this and I apologize for the late response.
> 
> That's a very good point. We'll look into changing our build pipelines so the `telegraf-operator` image is multi-arch. If not, we'll look into adding `nodeSelector`.

@pokebadgerswithspoon we have just released telegraf-operator v1.3.1 with multi-arch images supported.

The list of platforms is identical with one for `telegraf`.

Could you check that it works for you on all platforms if you use `quay.io/influxdb/telegraf-operator:v1.3.1` as the image so we can consider the issue closed?
